src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Stream.cs

```diff
@@ -1281,6 +1285,8 @@ protected override void Dispose(bool disposing)
                         return;
                     }
 
+                    GC.SuppressFinalize(this);
```

Comment:
`Http2ReadStream` already had a finalizer, I just added the missing `SuppressFinalize` to Dispose

Reply:
Stream.Dispose suppresses the finalizer.

Reply:
I see, this will be called by a call to Dispose(), not directly.
Will remove it 👍